plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java

```diff
@@ -136,6 +136,7 @@ private HttpClient getHttpClient() {
                 // this affects only the single request
                 .setRetryHandler(new DefaultHttpRequestRetryHandler(DEFAULT_MAX_RETRIES, true))
                 .setConnectionManager(new PoolingHttpClientConnectionManager())
+                .setUserAgent("Plugin Manager")
```

Comment:
Should ideally include the version as well

probably use the artifact id of the repo not plugin manager too

Reply:
Yeah that will be better. This way values will not be hardcoded, I will commit it soon.